repository/l.json

```diff
@@ -465,6 +465,19 @@
 				}
 			]
 		},
+		{
+			"name": "Lazy Unittester",
+			"author": "Connor Philip",
+			"description": "Creates Python unittest stubs",
+			"details": "https://github.com/connor-philip/lazy_unit_tester",
+			"labels": ["Python", "Unittesting"],
```

Comment:
You'll want to change the label to `python` lowercase, since all but 1 package is under that label. You probably want `Unittesting` to be `testing`: https://packagecontrol.io/browse/labels/testing.